lib/msal-common/src/telemetry/performance/PerformanceEvent.ts

```diff
@@ -94,6 +94,22 @@ export enum PerformanceEvents {
      * Used to acquire a token from Native component when native brokering is enabled.
      */
     NativeInteractionClientAcquireToken = "nativeInteractionClientAcquireToken",
+
+    /**
+     * Used to measure the time taken for completing embedded-broker handshake (PW-Broker).
+     */
+    BrokerHandhshake = "brokerHandshake",
```

Comment:
I assume the corresponding code is added in the PWB code related PR in the 1P library?

Reply:
Yes !

Reply:
Yep.